packages/core/src/utils.ts

```diff
@@ -178,8 +178,12 @@ export function isPercentage(value: unknown) {
   return typeof value === 'number' && value >= 0 && value <= 100
 }
 
+/**
+ * Get the time since the navigation was started.
+ *
+ * Note: this does not use `performance.timeOrigin` because it doesn't seem to reflect the actual
+ * time on which the navigation has started: it may be much farther in the past, at least in Firefox 71.
```

Comment:
What about linking the corresponding issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1429926?